data/css/layer.css

```diff
@@ -23,9 +26,16 @@
 .layer.even,
 .layer.even:focus,
 .layer.even:selected {
+    border: 1px solid shade (@bg_color, 1);
     background-color: shade (@bg_color, 1);
 }
 
+/* Hovered */
+.layer.hovered,
+.layer.even.hovered {
+  border: 1px solid #41c9fd;
```

Comment:
define a variable for this color since it's the primary hovering color we'll be using throughout the app.

Reply:
Done